src/components/chart/example-line-chart.njk

```diff
@@ -154,7 +154,9 @@
                 'Aug 2024',
                 'Sep 2024',
                 'Oct 2024'
-            ]
+            ],
+            "tickIntervalDesktop": 7,
```

Comment:
As noted in my response to Jane we might want to use some different examples here to try and avoid the ellipses as much as possible.